.circleci/config.yml

```diff
@@ -624,6 +627,7 @@ jobs:
           command: |
             export RELEASE_BRANCH_PATTERN='^v[0-9]+-release$'
             export TEST_BRANCH_PATTERN="^testflow[0-9]+$"
+            export PRE_BRANCH_PATTERN="^premainnet-[0-9|.]+$"
```

Comment:
I also decided to pre-emptively support premainnet for docker image production in git (I'll need this Monday morning).   Easier to test out over the weekend.